benchmarks/CMakeLists.txt

```diff
@@ -24,7 +24,8 @@ endmacro (do_benchmark)
 do_benchmark (large)
 do_benchmark (cmp)
 do_benchmark (createkeys)
-if (ENABLE_OPTIMIZATIONS)
+# exclude mingw
+if (ENABLE_OPTIMIZATIONS AND NOT WIN32)
```

Comment:
Its better to force ENABLE_OPTIMIZATIONS to OFF on WIN32, otherwise users might be confused.

Reply:
Why should users be confused, the only thing win32 users do not have are the benchmarks. Maybe the comment could say a little bit more.

Reply:
Yes, seems like I was confused here :-) Yes, then the solution is appropriate.